garrysmod/gamemodes/base/gamemode/animations.lua

```diff
@@ -54,11 +54,7 @@ function GM:HandlePlayerDucking( ply, velocity )
 
 	if ( !ply:IsFlagSet( FL_ANIMDUCKING ) ) then return false end
 
-	if ( velocity:Length2DSqr() > 0.25 ) then
-		ply.CalcIdeal = ACT_MP_CROUCHWALK
-	else
-		ply.CalcIdeal = ACT_MP_CROUCH_IDLE
-	end
+	ply.CalcIdeal = (velocity:Length2DSqr() > 0.25) and ACT_MP_CROUCHWALK or ACT_MP_CROUCH_IDLE

```

Comment:
I heard ternary operators dont exist in GLua, so its the same for the interpreter and the old version is more readable IMO.

Reply:
ah. okey. i did not know.